server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -2937,13 +2950,13 @@ public void testSelectBestPoolsUsageMatchShouldOverrideAllOtherSyspurposeMismatc
         // --- No satisfied syspurpose attributes on the consumer ---
 
         Product prod1 = createSysPurposeProduct(null, "mismatched_role", "mismatched_addon",
-            "mismatched_sla", "Production");
+            "mismatched_sla", "Production", null);
```

Comment:
The consumer should have a service_type set, and this should be "random_service_type" (that mismatches with the consumer's)

Reply:
Done.